pybossa/api/__init__.py

```diff
@@ -285,6 +286,19 @@ def _retrieve_new_task(project_id):
                       user=user_id_or_ip)
     return task, project.info.get('timeout'), handler
 
+def _guidelines_updated(project_id, user_id):
+    """Function to determine if guidelines has been
+        updated since last submission"""
+
+    query_attrs_log = dict(project_id=project_id, attribute='task_guidelines', desc=True)
+    query_attrs_task_run = dict(project_id=project_id, user_id=user_id)
+
+    guidelines_log = auditlog_repo.filter_by(limit=1, **query_attrs_log)
+    last_guidelines_update = dateutil.parser.parse(guidelines_log[0].created) if guidelines_log else None
+    task_runs = task_repo.filter_task_runs_by(limit=1, desc=True, **query_attrs_task_run)
+    last_task_run_time = dateutil.parser.parse(task_runs[0].created) if task_runs else None
+
+    return last_task_run_time < last_guidelines_update if last_task_run_time and last_guidelines_update else False
```

Comment:
this can be simplified as,
```python
last_guidelines_update = dateutil.parser.parse(guidelines_log[0].created) if guidelines_log else 0
last_task_run_time = dateutil.parser.parse(task_runs[0].created) if task_runs else 0
return last_task_run_time < last_guidelines_update
```
Also, is there possibility to get notified multiple times?